src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/ReviewCustomTx/index.tsx

```diff
@@ -132,7 +134,7 @@ const ReviewCustomTx = ({ onClose, onPrev, tx }: Props): React.ReactElement => {
           </Paragraph>
         </Row>
         <Row align="center" margin="md">
-          <Img alt="Ether" height={28} onError={setImageToPlaceholder} src={getEthAsToken('0').logoUri} />
+          <TokenLogo height={28} tokenName={token?.name} tokenLogoUri={token?.logoUri} />
```

Comment:
So in the other place we check if `token` (which I would rename to `ethToken` or similar, to show that it is always eth) is `null` here we kind of assume that it is not, I would try to be consistent